eng/InstallRuntimes.proj

```diff
@@ -0,0 +1,109 @@
+     $(MicrosoftNETCoreApp31Version) $(MicrosoftAspNetCoreApp31Version)  - 3.1 version
+     $(MicrosoftNETCoreApp30Version) $(MicrosoftAspNetCoreApp30Version)  - 3.0 version
+     $(MicrosoftNETCoreApp21Version) $(MicrosoftAspNetCoreApp21Version)  - 2.1 version
+
+     From Arcade:
+
+     $(DotNetRoot) - the SDK/runtime installation root
+     $(RepositoryEngineeringDir) - the "eng" directory
+     $(VersionsPropsPath) - path of Versions.props
+  -->
+
+  <PropertyGroup>
+    <DailyTest Condition="'$(DailyTest)' == ''">false</DailyTest>
+    <BuildArch Condition="'$(BuildArch)' == ''">$(Platform)</BuildArch>
+    <BuildArch Condition="'$(BuildArch)' == ''">x64</BuildArch>
+    <CommonInstallArgs>-architecture $(BuildArch)</CommonInstallArgs>
+    <TestConfigFileName Condition="'$(BuildArch)' != 'x86'">$(DotNetRoot)Debugger.Tests.Versions.txt</TestConfigFileName>
+    <TestConfigFileName Condition="'$(BuildArch)' == 'x86'">$(DotNetRoot)x86\Debugger.Tests.Versions.txt</TestConfigFileName>
+  </PropertyGroup>
+
+  <Choose>
+    <When Condition="$([MSBuild]::IsOsPlatform(Windows))">
+      <PropertyGroup>
+        <PowershellWrapper>powershell -NonInteractive -ExecutionPolicy ByPass -NoProfile -command</PowershellWrapper>
+        <DotnetInstallScriptCmd>'$(RepositoryEngineeringDir)common\dotnet-install.ps1'</DotnetInstallScriptCmd>
+      </PropertyGroup>
+    </When>
+    <Otherwise>
+      <PropertyGroup>
+        <DotnetInstallScriptCmd>$(RepositoryEngineeringDir)common/dotnet-install.sh</DotnetInstallScriptCmd>
+      </PropertyGroup>
+    </Otherwise>
+  </Choose>
+
+  <ItemGroup>
+    <!-- We always test on latest, so install that one even for scheduled builds -->
+    <TestVersions Include="Latest" RuntimeVersion="$(MicrosoftNETCoreAppVersion)" AspNetVersion="$(MicrosoftAspNetCoreAppRefVersion)" Install="true" />
+    <TestVersions Include="31" RuntimeVersion="$(MicrosoftNETCoreApp31Version)" AspNetVersion="$(MicrosoftAspNetCoreApp31Version)" Install="!$(DailyTest)" />
+    <TestVersions Include="30" RuntimeVersion="$(MicrosoftNETCoreApp30Version)" AspNetVersion="$(MicrosoftAspNetCoreApp30Version)" Install="!$(DailyTest)" />
+    <TestVersions Include="21" RuntimeVersion="$(MicrosoftNETCoreApp21Version)" AspNetVersion="$(MicrosoftAspNetCoreApp21Version)" Install="!$(DailyTest)" />
+  </ItemGroup>
+
+  <Target Name="InstallTestRuntimes" 
+          BeforeTargets="RunTests"
+          DependsOnTargets="InstallRuntimesWindows;InstallRuntimesUnix;WriteTestVersionManifest" />
+
+  <Target Name="InstallRuntimesWindows"
+          Condition="$([MSBuild]::IsOsPlatform(Windows))"
+          Inputs="$(VersionsPropsPath)" Outputs="$(TestConfigFileName);$(DotNetRoot)\shared">
+
+    <!-- Install SDK for the x86 architecture -->
+    <Exec Command="$(PowershellWrapper) &quot;&amp; { &amp;$(DotnetInstallScriptCmd) $(CommonInstallArgs) -version $(NETCoreSDKVersion) -runtime '' }&quot;"
```

Comment:
Remove the runtime argument in this case, but why do you need this?

Reply:
This is installing the x86 SDK under the .dotnet/x86 directory that the arcade testing is expecting.

Reply:
The question is why do we need an SDK. Isn't this only to run?

Reply:
Arcade only installs the x64 SDK on windows and that is used to build.  For testing x86, the x86 SDK needs to be installed and the arcade dotnet-install.ps1 installs it in .dotnet/x86.